coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -1406,13 +1406,22 @@ func (s *server) CompleteJob(ctx context.Context, completed *proto.CompletedJob)
 
 		err = s.Database.UpdateTemplateVersionExternalAuthProvidersByJobID(ctx, database.UpdateTemplateVersionExternalAuthProvidersByJobIDParams{
 			JobID:                 jobID,
-			ExternalAuthProviders: json.RawMessage(externalAuthProvidersMessage),
+			ExternalAuthProviders: externalAuthProvidersMessage,
 			UpdatedAt:             s.timeNow(),
 		})
 		if err != nil {
 			return nil, xerrors.Errorf("update template version external auth providers: %w", err)
 		}
 
+		err = s.Database.InsertTemplateVersionTerraformValuesByJobID(ctx, database.InsertTemplateVersionTerraformValuesByJobIDParams{
+			JobID:      jobID,
+			CachedPlan: jobType.TemplateImport.Plan,
+			UpdatedAt:  s.timeNow(),
```

Comment:
No need to fix this, but we call `s.timeNow()` a lot in this function. Why do we not just call it once, and reuse that time?